imports/plugins/included/product-detail-simple/client/components/priceRange.less

```diff
@@ -0,0 +1,11 @@
+.fa-twitter{
```

Comment:
This `.fa-twitter` and `.fa-facebook` have some similar styles. You could either do this
```css
.fa-twitter, .fa-facebook {
  padding-top: 15px;
  align-self: center;
  padding-top: 15px;
  font-size: 40px;
}

.fa-facebook {
  padding-left: 15px;
}
```

or you could take advantage of `less` extend and do this
```css
.social-button {
  padding-top: 15px;
  align-self: center;
  padding-top: 15px;
  font-size: 40px;
}

.fa-twitter {
&:extend(.social-button);
}

.fa-facebook {
  &:extend(.social-button);
  padding-left: 15px;
}
```

Either way, you would be keeping your css DRY